addons/op-badge/forums.css

```
@@ -1,6 +1,6 @@
 .sa-original-poster {
   display: inline-block;
-  background-color: var(--darkWww-button-scratchr2PostHeader, #28a5da);
+  background-color: var(--darkWww-button, #855cd6);
```

Only the default value needs to be updated here.

```suggestion
  background-color: var(--darkWww-button-scratchr2PostHeader, #855cd6);
```